assets/js/blocks/checkout/inner-blocks/checkout-payment-block/edit.tsx

```diff
@@ -9,6 +9,9 @@ import { ADMIN_URL, getSetting } from '@woocommerce/settings';
 import ExternalLinkCard from '@woocommerce/editor-components/external-link-card';
 import { innerBlockAreas } from '@woocommerce/blocks-checkout';
 import Noninteractive from '@woocommerce/base-components/noninteractive';
+import { GlobalPaymentMethod } from '@woocommerce/type-defs/payments';
```

Comment:
```suggestion
import { GlobalPaymentMethod } from '@woocommerce/types';
```

With https://github.com/woocommerce/woocommerce-blocks/pull/8019, I created a PR to simplify the `@woocommerce/types` imports.